content/master/snippets/composition-environments/claim.yaml

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
is there a better/new place to put these examples? I think with the [EnvironmentConfig PR](https://github.com/crossplane/crossplane/pull/3007), these snippets were added with the intention to serve as examples, but they aren't actually being displayed anywhere.

Is there a better place where we can display examples like these for users?

Reply:
If it's just yaml files, it may make the most sense somewhere in the c/c repo.

If we want to add the content of the yaml files to a docs page with context then the [Knowledge Base/Guides](https://deploy-preview-271--crossplane.netlify.app/knowledge-base/guides/) or even a new section in the KB called "Examples" or "Samples" would be a good home.

Reply:
Good point @plumbis - looks like there is some documentation on how to use the Environment related patch types in concepts/composition/#patch-types, so i think we're good here for getting the release out.

I'll open a follow-up issue to author a more complete end to end guide on how to use EnvironmentConfig - that will still be useful to teach people more completely how to use them.

Reply:
#305 opened to track this EnvironmentConfig user guide :)